bin/snapcraft

```diff
@@ -14,23 +14,4 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-import os
-import sys
-
-# make running from bzr snapshot easy
-if getattr(sys, 'frozen', False):
-	topdir = os.path.dirname(sys.executable)
-else:
-	topdir = os.path.abspath(os.path.join(__file__, '..', '..'))
-	
-if os.path.exists(os.path.join(topdir, 'setup.py')):
-    sys.path = [topdir] + sys.path
-
-
-if __name__ == '__main__':
-    import snapcraft.cli
-    import snapcraft.internal.dirs
-    snapcraft.internal.dirs.setup_dirs()
-    snapcraft.cli.run(prog_name='snapcraft',
-                      obj=dict(project=snapcraft.ProjectOptions()))
+__import__('snapcraft.cli.__main__', fromlist=['__name__'], level=0)
```

Comment:
Why are you using this instead of a normal import?

Also, the __import__ docs say: "Direct use of __import__() is also discouraged in favor of importlib.import_module()"

Reply:
I replicated `pkg_resources`

Reply:
it is now a comment in the code as well